templates/patterns/social_media_links/social_media_links.ui_patterns.yml

```diff
@@ -0,0 +1,32 @@
+social_media_links:
+  label: "Social media links"
+  description: "Social network icon with link of label."
```

Comment:
This really means nothing, lets put something more meaningful now that we can: "Showcases a list of social media links with their respective icons."